### PR TITLE
use travis docker infra

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-        - TOX_ENV=py32
-        - TOX_ENV=py33
-        - TOX_ENV=py34
-        - TOX_ENV=pypy
-        - TOX_ENV=docs
-        - TOX_ENV=pep8
-        - TOX_ENV=py3pep8
+matrix:
+    include:
+    - env: TOXENV=py26
```
Comment (alex) [inline .travis.yml]: These should all be indented

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
+        - env: TOXENV=pep8
+        - env: TOXENV=py3pep8
 
 install:
     - pip install coveralls tox
-    - sudo apt-get install libenchant-dev
 
 script:
-    - tox -e $TOX_ENV
+    - tox
```
Comment (sigmavirus24) [inline .travis.yml]: Does tox now pick up `TOXENV` automatically?

Comment (reaperhulk) [inline .travis.yml]: yep: http://tox.readthedocs.org/en/latest/example/general.html?highlight=toxenv

Comment (sigmavirus24) [inline .travis.yml]: :cake:

**Pull request merged**